Paper.tex

```diff
@@ -338,7 +337,20 @@ \subsection{The Transaction} \label{subsec:transaction}
 $T_{\mathrm{w}} = 27 + T_{\mathrm{y}}$ or $T_{\mathrm{w}} = 2\beta + 35 + T_{\mathrm{y}}$ (see EIP-155 by \cite{EIP-155}).
 \end{description}
 
-Additionally, a contract creation transaction (regardless whether legacy or EIP-2930) contains:
+There are differences in how one's acceptable gas price is specified in type 2 transactions versus type 0 and type 1 transactions. Type 2 transactions take better advantage of the gas market improvements introduced in EIP-1559 by explicitly limiting the \textit{priority fee}\footnote{The \textit{priority fee} is discussed in greater detail in sections \ref{ch:payment} and \ref{ch:transactions}.} that is paid. Type 2 transactions have the following two fields related to gas:
+
+\begin{description}
+  \item[maxFeePerGas]\linkdest{max_fee_per_gas}{} A scalar value equal to the maximum number of Wei to be paid per unit of \textit{gas} for all computation costs incurred as a result of the execution of this transaction; formally $T_{\mathrm{m}}$.
+  \item[maxPriorityFeePerGas]\linkdest{max_priority_fee_per_gas}{} A scalar value equal to the maximum number of Wei to be paid to the block's fee recipient as an incentive to include the transaction; formally $T_{\mathrm{f}}$.
+\end{description}
+
+In contrast, type 0 and type 1 transactions specify gas price as a single value:
+
+\begin{description}
+  \item[gasPrice]\linkdest{tx_gas_price_T__p}{} A scalar value equal to the number of Wei to be paid per unit of \textit{gas} for all computation costs incurred as a result of the execution of this transaction; formally $T_{\mathrm{p}}$.
```

Comment:
It'd be good to note that for type 0 & 1 transactions effectively Tm = Tf = Tp.

Reply:
That's not precisely accurate to say for Tf I think, since priority fee is derived from gas price and the base fee.

For type 0 & 1 transactions:
`Tf = Tp - Hf`

But you're right to say Tp is analogous to Tm, but it's a guarantee and not a max.

Reply:
I was going back through comments, this is correct to say (Tm = Tf = Tp), and that's how the EIP handles type 0 and 1 via `normalize_transaction`. I added to a footnote to this effect.

![image](https://github.com/ethereum/yellowpaper/assets/13876978/ebbf49e2-085b-4e65-91a7-25ce9676ae4b)